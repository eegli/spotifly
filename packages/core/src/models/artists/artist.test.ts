import { authProvider_WILL_HIT_API } from '@/test/setup';
import { Artists } from '.';

describe('Artists', () => {
  test('works', async () => {
    const artists = new Artists(authProvider_WILL_HIT_API);
    const res = await artists.ids(['12Chz98pHFMPJEknJQMWvI']).get();
    expect(res).toMatchSnapshot();
  });
});
