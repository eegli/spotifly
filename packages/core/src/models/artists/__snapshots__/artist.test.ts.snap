// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artists works 1`] = `
[
  {
    "external_urls": {
      "spotify": "https://open.spotify.com/artist/12Chz98pHFMPJEknJQMWvI",
    },
    "followers": {
      "href": null,
      "total": 7049247,
    },
    "genres": [
      "modern rock",
      "permanent wave",
      "rock",
    ],
    "href": "https://api.spotify.com/v1/artists/12Chz98pHFMPJEknJQMWvI",
    "id": "12Chz98pHFMPJEknJQMWvI",
    "images": [
      {
        "height": 640,
        "url": "https://i.scdn.co/image/ab6761610000e5eb02fffd59f8dd3d45b2e7ef30",
        "width": 640,
      },
      {
        "height": 320,
        "url": "https://i.scdn.co/image/ab6761610000517402fffd59f8dd3d45b2e7ef30",
        "width": 320,
      },
      {
        "height": 160,
        "url": "https://i.scdn.co/image/ab6761610000f17802fffd59f8dd3d45b2e7ef30",
        "width": 160,
      },
    ],
    "name": "Muse",
    "popularity": 74,
    "type": "artist",
    "uri": "spotify:artist:12Chz98pHFMPJEknJQMWvI",
  },
]
`;
